### ISSUE TITLE
Stamina call back error

### ISSUE THREAD

**Issue by nPHYN1T3** [minetest-LOTR/Lord-of-the-Test#156]
2020-06-13 10:55:19: WARNING[Server]: WARNING: minetest.setting_* functions are deprecated.  Use methods on the minetest.settings object.
2020-06-13 10:55:19: ERROR[Main]: ServerError: AsyncErr: ServerThread::run Lua: Runtime error from mod 'stamina' in callback on_joinplayer(): ...etest/games/Lord-of-the-Test-1.2.2/mods/stamina/init.lua:31: attempt to call method 'get_meta' (a nil value)
2020-06-13 10:55:19: ERROR[Main]: stack traceback:
2020-06-13 10:55:19: ERROR[Main]: 	...etest/games/Lord-of-the-Test-1.2.2/mods/stamina/init.lua:31: in function 'stamina_read'
2020-06-13 10:55:19: ERROR[Main]: 	...etest/games/Lord-of-the-Test-1.2.2/mods/stamina/init.lua:327: in function <...etest/games/Lord-of-the-Test-1.2.2/mods/stamina/init.lua:323>
2020-06-13 10:55:19: ERROR[Main]: 	/usr/share/games/minetest/builtin/game/register.lua:412: in function </usr/share/games/minetest/builtin/game/register.lua:392>
2020-06-13 10:55:19: ACTION[Server]: singleplayer leaves game. List of players: 
2020-06-13 10:55:19: WARNING[Server]: WARNING: minetest.setting_* functions are deprecated.  Use methods on the minetest.settings object.

**Comment by Amaz1** [comment]
I can't reproduce this. When does this happen? What version of Minetest are you using?

**Comment by nPHYN1T3** [comment]
Sorry I didn't add that stuff. v0.4.16 Xubuntu 18.04.4 Happens on launch. i.e. it won't launch. However I just downloaded it again and installed and it seems to work. The problem here (my fault for sure) is I'm not sure if minetest might have been updated since i posted this and now or if the game was. I tried a bunch of games a while back, most were broken for me. I just kinda wrote things off as some where version conflicts with my minetest build due to age or abandonment.
